### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by Mehrm0**
### Description

Added missing live sample to the \<data\> element and a missing title to an example in <a>.

### CODE REVIEW

File: files/en-us/web/html/element/data/index.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ The following example displays product names but also associates each name with
   <li><data value="400">Mega Jumbo Ketchup</data></li>
 </ul>
 ```
+### Result
```
Comment (dawei-wang) [inline files/en-us/web/html/element/data/index.md:90]:
```suggestion
### Result

```

**Review by dawei-wang — COMMENTED** [review]
Fix linting

File: files/en-us/web/html/element/data/index.md [hunk 2]
```diff
@@ -87,6 +87,9 @@ The following example displays product names but also associates each name with
...
```
Comment (dawei-wang) [inline files/en-us/web/html/element/data/index.md:90]:
```suggestion

### Result

```

**Review by dawei-wang — COMMENTED** [review]
linting

**Comment by Mehrm0** [comment]
> linting

I commited the suggestion, is it enough or should I fix it somewhere else?

**Comment by dawei-wang** [comment]
There seems to be too many blank lines now. There should be one blank line directly before and one directly after the `##Result` heading. It may be easier to go back to the branch in your fork to fix it.

**Comment by Mehrm0** [comment]
> There seems to be too many blank lines now. There should be one blank line directly before and one directly after the `##Result` heading. It may be easier to go back to the branch in your fork to fix it.

Thanks @dawei-wang , I pushed a new branch and created a new pull request: https://github.com/mdn/content/pull/25067

**Pull request closed without merging**